tests/test_config.py

```diff
@@ -13,25 +13,65 @@
 # Tests that a ProjectConfig object can be created with valid parameters
 def test_valid_parameters():
     project_config = ProjectConfig(dbt_project_path="path/to/dbt/project")
-    assert project_config.dbt_project_path == Path("path/to/dbt/project")
+    assert project_config.parsed_dbt_project_path == Path("path/to/dbt/project")
     assert project_config.models_relative_path == Path("path/to/dbt/project/models")
     assert project_config.seeds_relative_path == Path("path/to/dbt/project/seeds")
     assert project_config.snapshots_relative_path == Path("path/to/dbt/project/snapshots")
     assert project_config.manifest_path is None
 
 
-def test_init_with_manifest():
+# Since dbt_project_path is now an optional parameter, we should test each combination for init and validation
+
+
+# Passing a manifest AND project together should succeed, as previous
```

Comment:
I think there are two better alternatives to comments in this case:
- improve the test method name (`test_init_with_manifest_and_project_succeeds`), or
- add this comment as a docstring of the test function

Happy with either of the alternatives! This applies to most of the tests added

Reply:
For this one, i have done a combination of both - preserved the comments as docstrings, and tried to standardize the function names so that they are more clear. Let me know if this is what you expected :)

Reply:
Thank you very much!